modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,38 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting the {sandboxed-containers-operator} deployment custom resource
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} 1.0.0 installed.
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
+. Click *Delete* in the confirmation window.
+. Delete all `pod`, `deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, and `StatefulSet` objects that use `kata` as `runtimeClass`.
+.. Use the OpenShift CLI (oc) to retrieve all pods that use `kata` as `runtimeClassName` by running the following command:
```

Comment:
I think this should be a note not a sub-step for example "You can use .... to get all pods using the 'kata' runtimeClass...

Reply:
@jensfr Shouldn't the above set of instructions _precede_ the removal of the kataconfig?